client/js/table-model.js

```diff
@@ -13,6 +13,14 @@ class TableModel {
     setValue(location, value) {
         this.data[this._getCellId(location)] = value;
     }
+    getColumnValues(column) {
+        const values = [];
+        for (let i = 0; i < this.numRows; i++) {
```

Comment:
Elsewhere you use "col" in a for loop through columns. For consistency (and because it's easier to read), use "row" here instead of the bland i.